packages/components/color-picker/src/composables/use-alpha-slider.ts

```diff
@@ -62,11 +67,38 @@ export const useAlphaSlider = (props: AlphaSliderProps) => {
     }
   }
 
+  function handleKeydown(event: KeyboardEvent) {
+    const step = 1
```

Comment:
Do you think we need to expose this step and allow users to set it themselves?

Reply:
Adding the step attribute may affect the exact value entered by the user using the keyboard.

Reply:
Okay, let's keep things as they are.